### DISCUSSION TITLE
Availability of macOS Ventura 13.3.1

### DISCUSSION THREAD

**Discussion by tsimbalar** [codemagic-ci-cd/codemagic-docs#1622] [Q&A] [answered]
It seems like macOS Ventura 13.3.1 introduces changes in the rendered Flutter Golden Tests ... 

Some developers already have upgraded their machines to that version, but then all the goldens they generate on their machine will fail in CodeMagic, because the rendering will be slightly different.

Is their an expected date for availability of machines with macOS 13.3.1 ?

Thanks

**Comment by icarusdust** [comment, marked as answer]
@tsimbalar, we have already discussed it in Intercom but let me mention it here as well that 13.3.1 is already available.

**Comment by suyangdong** [comment]
Are there any difference between 13.3.1 and 13.4.1+?
I have golden tests that run fine on both 13.4.1 and 13.5 with less than 0.02% of difference from each other. 
However there are more than 1% of difference when running on the 13.3.1 (CodeMagic)
